src/api/public/apidocs-new/paths/source_project_name_package_name_cmd_diff.yaml

```diff
@@ -0,0 +1,173 @@
+post:
+  summary: Returns the source diff of a package.
+  description: |
+    Without parameters, return the diff of the last change made to the package. This command doesn't perform any action.
+  security:
+    - basic_authentication: []
+  parameters:
+    - $ref: '../components/parameters/project_name.yaml'
+    - $ref: '../components/parameters/package_name.yaml'
+    - in: query
+      name: cacheonly
+      schema:
+        type: string
+      description: |
+        Set to `1` to retrieve a diff only if the diff is already cached.
+
+        If the diff is not cached return a 412 "Precondition failed" error.
+      example: 1
+    - in: query
+      name: expand
+      schema:
+        type: string
+      description: Set to `1` to expand any link before diffing.
+      example: 1
+    - in: query
+      name: file
+      schema:
+        type: string
+      description: Limit diff to the given file name.
+      example: hello_world.spec
+    - in: query
+      name: filelimit
+      schema:
+        type: string
+      description: |
+        Limit the diff size by line numbers.
+
+        By default it takes a built-in default value.
```

Comment:
Would be nice to know what that built-in default value is...

Reply:
...200. Changed.